emu/processes/wps_geospatial.py

```diff
@@ -0,0 +1,137 @@
+"""
+Process using an application/geo+json and 'image/tiff; application=geotiff' input. Used to test UI interactions.
```

Comment:
Please clarify which interactions you are testing exactly. I still don't understand why we'd need to be making geospatial operations inside Emu. 
Testing a BoundingBox Complex Input would be valuable, but this is not high on our priority list at the moment.

Reply:
I can remove this now that I have `wps_geodata.py`